frontend/assets/javascripts/src/main.js

```diff
@@ -84,7 +77,6 @@ require([
     (new JoinFree()).init();
     (new PaidForm()).init();
     (new StaffForm()).init();
-    (new Thankyou()).init(header);
```

Comment:
Nice! Now that `Thankyou` is gone we can also update the Header initialisation to be `(new Header()).init();` as we no longer need to pass the header instance around (which is good)